source/Ship.cpp

```diff
@@ -1718,7 +1722,8 @@ shared_ptr<Ship> Ship::Board(bool autoPlunder, bool nonDocking)
 	hasBoarded = false;
 
 	shared_ptr<Ship> victim = GetTargetShip();
-	if(CannotAct(Ship::ActionType::BOARD) || !victim || victim->IsDestroyed() || victim->GetSystem() != GetSystem())
+	if(CannotAct(Ship::ActionType::BOARD) || !victim || victim->IsRemoved()
+		|| victim->GetSystem() != GetSystem())
```

Comment:
Can't this stay on one line?
```suggestion
	if(CannotAct(Ship::ActionType::BOARD) || !victim || victim->IsRemoved() || victim->GetSystem() != GetSystem())
```